taskt/Core/Automation/Commands/Word Commands/WordAppendTextCommand.cs

```diff
@@ -1,137 +1,145 @@
-        [Attributes.PropertyAttributes.SampleUsage("Hello World or [vText]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Text")]
+        [InputSpecification("Enter the text to append to the Document.")]
+        [SampleUsage("Hello World || {vText}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_TextToSet { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Select or Enter the text font name")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Arial")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Calibri")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Helvetica")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Times New Roman")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Verdana")]
-        [Attributes.PropertyAttributes.InputSpecification("Specify the font name.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select **Arial**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Font Name")]
+        [PropertyUISelectionOption("Arial")]
+        [PropertyUISelectionOption("Calibri")]
+        [PropertyUISelectionOption("Helvetica")]
+        [PropertyUISelectionOption("Times New Roman")]
+        [PropertyUISelectionOption("Verdana")]
+        [InputSpecification("Select or provide a valid font name.")]
+        [SampleUsage("")]
+        [Remarks("")]
         public string v_FontName { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Select or Enter the text font size")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("10")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("11")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("12")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("14")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("16")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("18")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("20")]
-        [Attributes.PropertyAttributes.InputSpecification("Specify the font name.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select **14**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Font Size")]
+        [PropertyUISelectionOption("10")]
+        [PropertyUISelectionOption("11")]
+        [PropertyUISelectionOption("12")]
+        [PropertyUISelectionOption("14")]
+        [PropertyUISelectionOption("16")]
+        [PropertyUISelectionOption("18")]
+        [PropertyUISelectionOption("20")]
+        [InputSpecification("Select or provide a valid font size.")]
```

Comment:
Similarly, It should be "Select a font size."

Reply:
Same here. Font number can be anything.

Reply:
got it